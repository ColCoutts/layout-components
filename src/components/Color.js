import React from 'react';


export default function Color() {
  const color = {
    name: 'Red',
    hex: 'FF6347',
    rgb: '255,99,71'
  };

  const divStyle = {
    color: 'blue'
  };

  const dtStyle = {
    color: 'green'
  }
  
  return (
    <div style={divStyle}>
      <dl>
        <dt style={dtStyle}>Name</dt>
        <dd>{color.name}</dd>

        <dt>Hex</dt>
        <dd>{color.hex}</dd>

        <dt>RGB</dt>
        <dd>{color.rgb}</dd>
      </dl>
    </div>
  );
}
