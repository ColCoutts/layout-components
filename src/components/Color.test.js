import React from 'react';
import { shallow } from 'enzyme';
import Color from './Color';

describe('color snapshots', () => {
  it('renders correct html', () => {
    const wrapper = shallow(<Color />);
    expect(wrapper).toMatchSnapshot();
  });
});
