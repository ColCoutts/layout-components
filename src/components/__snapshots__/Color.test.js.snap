// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`color snapshots renders correct html 1`] = `
<div
  style={
    Object {
      "color": "blue",
    }
  }
>
  <dl>
    <dt
      style={
        Object {
          "color": "green",
        }
      }
    >
      Name
    </dt>
    <dd>
      Red
    </dd>
    <dt>
      Hex
    </dt>
    <dd>
      FF6347
    </dd>
    <dt>
      RGB
    </dt>
    <dd>
      255,99,71
    </dd>
  </dl>
</div>
`;
